Source/Bogus.Tests/LocaleTests/PtLocale.cs

```diff
@@ -38,7 +38,7 @@ public void name_tests()
       [Fact]
       public void date_tests()
       {
-         var d = new Date("pt_PT");
+         var d = new Date(locale: "pt_PT");
```

Comment:
also, you'll need to revert this change if it's not directly related to fixing the problem at hand.

superficial changes only create noise in the PR review/merge (and in the unlikely event, revert) process. PR change sets into this repo should only include the minimal amount of changes required to fix or correct a single problem.

coding style improvements would need to go under a different PR.

Reply:
I did only do the necessary changes. Maybe thats unintentional and a leftover from the pwssing a function over a second ctor.  I will chech it. thanks for the hint.

Reply:
Is reverted.